### PR TITLE
Create edit profile action

### PR DESCRIPTION
**Pull request by gpkbluesky**
screenshot from 2018-07-27 15-53-50

### CODE REVIEW

**Comment by gpkbluesky** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -18,6 +18,22 @@ def check_access
...
+  def check_edit_authority
+    if @user.supervisor?
+      return if current_user.admin?
+    end
+    if @user.trainee?
+      return if current_user.admin? || current_user.supervisor?
+    end
+    return if current_user? @user
+    flash[:warning] = t "flash.access_denied"
+    redirect_to root_path
```
Comment (phamvanbk11) [inline app/helpers/sessions_helper.rb]: dùng case when đi em

**Comment by phamvanbk11** [comment]
commented

**Comment by gpkbluesky** [comment]
ready

**Pull request merged**